djmoney/__init__.py

```diff
@@ -1,2 +1,2 @@
-__version__ = "1.0"
+__version__ = "1.1"
```

Comment:
I'd rather avoid setting version in pull requests - the release date might be different and also we can add more features after this one

Reply:
Just ranting here...

I agree that PRs should contain version changes, those we handle separately - also a 1.1 release would be misleading if it only contains a bug fix. I think when doing minor version bumps, the dependency wouldn't traverse very far, and people would feel obliged to read release notes looking for breakage... and of course one doesn't do that when there isn't time.

So ultimately, a minor version bump can be in the interest of non, if it isn't substantial :)